### ISSUE TITLE
there's apk in the Code Area

### ISSUE THREAD

**Issue by AngelShadow2017** [AngelShadow2017/pvztr_weiduan#1]

**Comment by dimaspaf14** [comment]
How to install?
Me android 6.0

**Comment by AngelShadow2017** [comment]
> How to install? Me android 6.0

I am sorry to hear that:)

you can just click the link and download

**Comment by AngelShadow2017** [comment]
![Screenshot_20220211_142332_com android chrome](https://user-images.githubusercontent.com/32807696/153546206-c57a956e-f9e5-4339-91a0-3f6202cb9d33.jpg)
did you see android 1.10.5 ? click it and download

**Comment by AngelShadow2017** [comment]
if you cant open the app, you need to ensure your Android System Webview in the latest version, if not, you can update in Google Play first.

**Comment by dimaspaf14** [comment]
Ok thanks, but im stuck at chinese qq website, idk why

**Comment by AngelShadow2017** [comment]
> Ok thanks, but im stuck at chinese qq website, idk why

oh if your app tips to add qq group. dont need to mind, just update your Android system webview

**Comment by AngelShadow2017** [comment]
find it in google play

**Comment by dimaspaf14** [comment]
Done updating, now it works, thanks

**Comment by AngelShadow2017** [comment]
> Done updating, now it works, thanks

Enjoy our game

**Comment by AngelShadow2017** [comment]
> Done updating, now it works, thanks

and what's the experience?

We are committed to translate it to English. But for some reason, it is some kind of hard work.

**Comment by dimaspaf14** [comment]
Nice game, some resources still chinese, but rest of em are english

**Comment by AngelShadow2017** [comment]
> Nice game, some resources still chinese, but rest of em are english

that's true, you can also join our discord https://discord.gg/5UZggBD3Uk

**Comment by dimaspaf14** [comment]
Done
